site/content/3.10/arangograph/organizations/_index.md

```diff
@@ -35,18 +35,29 @@ member of one _Free-to-try_ tier organization at a time.
 With the ArangoGraph Insights Platform, your organization can choose one of the
 following packages.
 
-### Free Trial
+### Perpetual Free Trial
```

Comment:
Should this be "Perpetual Free Tier" or is it "Trial" on purpose to indicate that this isn't suitable for production?